installer/client/src/pages/AdminUserCreation.vue

```diff
@@ -0,0 +1,206 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <oxd-form
+    class="orangehrm-installer-page"
+    :loading="isLoading"
+    @submit-valid="onSubmit"
+  >
+    <oxd-text tag="h5" class="orangehrm-installer-page-title">
+      Admin User Creation
+    </oxd-text>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      Select the name, email address, username and password to create the admin
+      user for your OrangeHRM Instance
+    </oxd-text>
+    <br />
+    <oxd-grid :cols="4" class="orangehrm-full-width-grid">
+      <oxd-grid-item class="--span-column-2">
+        <oxd-input-group label="Employee Name" :classes="classes">
+          <oxd-input-field
+            v-model="adminUser.firstName"
+            :rules="rules.firstName"
+          />
+          <oxd-input-field
+            v-model="adminUser.lastName"
+            :rules="rules.lastName"
+          />
+        </oxd-input-group>
+      </oxd-grid-item>
```

Comment:
Need help with styles for this. Currently appearing like this:

![image](https://user-images.githubusercontent.com/97493184/165078244-b65890cb-f275-4e0b-9410-2e7696a2118f.png)

Need to look like this:
![image](https://user-images.githubusercontent.com/97493184/165078323-4de10732-c39c-4c11-bbaf-a58f289b1513.png)

Reply:
try nested grid:
```vue
<oxd-grid :cols="2" class="orangehrm-full-width-grid">
    <oxd-grid-item class="--span-column-2">
        <oxd-input-group label="Employee Name" :classes="classes">
          <oxd-input-field
            v-model="adminUser.firstName"
            :rules="rules.firstName"
          />
      </oxd-grid-item>
    <oxd-grid-item class="--span-column-2">
          <oxd-input-field
            v-model="adminUser.lastName"
            :rules="rules.lastName"
          />
        </oxd-input-group>
      </oxd-grid-item>
</oxd-grid>
```
you may have adjust `:cols="2" class="orangehrm-full-width-grid"`
`class="--span-column-2"`

Reply:
tried with a nested grid and some other variations but couldn't get it to work :frowning_face:

Reply:
got it working with a nested grid not sure if the approach I took is okay. if you compare between the name and password fields you can see it is still a little off.

![image](https://user-images.githubusercontent.com/97493184/165210534-e942e702-a63e-4742-b144-bcfddf3705ee.png)